### PR TITLE
Add Docker specific settings

### PR DESCRIPTION
**Pull request by CMeza99**
From https://github.com/moby/moby/blob/e2e5d4bc9da5ba17bb2822909611f8300fdd80f0/contrib/mkimage/debootstrap,
- setting dpkg unsafe-io
- allowing kernel autoremove

### CODE REVIEW

**Comment by andsens** [comment]
Nice job, especially with the references. I wouldn't add tasks from other plugins though (`CreateDpkgCfg`). I think it'd be much better to simply add a proper manifest example that causes that task to be added.

**Comment by CMeza99** [comment]
Yeah, using another plugin task like that felt dirty. So, I added at as a requirement. I also updated the docs and included two new manifest files for Docker.

File: bootstrapvz/plugins/minimize_size/__init__.py [hunk 1]
```diff
@@ -54,8 +54,8 @@ def resolve_tasks(taskset, manifest):
         if apt.get('autoremove_suggests', False):
             taskset.add(tasks.apt.AptAutoremoveSuggests)
     if 'dpkg' in manifest.plugins['minimize_size']:
-        filter_tasks = [tasks.dpkg.CreateDpkgCfg,
-                        tasks.dpkg.InitializeBootstrapFilterList,
+        taskset.add(tasks.dpkg.CreateDpkgCfg)
```
Comment (andsens) [inline bootstrapvz/plugins/minimize_size/__init__.py]: What's the motivation for this change?

Comment (CMeza99) [reply bootstrapvz/plugins/minimize_size/__init__.py]: The path for the dpkg config needs to be created so the setting `force-unsafe-io`, so I wanted to make sure the task CreateDpkgCfg was called regardless of is set in the manifest file for` minimize_size: { dpkg: {} }`. But that still seems hacky. So I have change to just creating the dir in the Docker provider.

**Comment by andsens** [comment]
The manifests & docs look great! Instead of using `makedirs` though, you should probably just move `CreatDpkgCfg` to `common/` so that both plugins may add and depend on it.  
After all that's partially why `common/` exists :-)  
The tasklist is a set, so there is no harm in adding a task twice.

**Comment by CMeza99** [comment]
updated :)

**Comment by andsens** [comment]
Perfect! Merging...
